Project/Engine/src/sysFrame/Object/Light/PointLight/PointLight.cpp

```
@@ -11,10 +14,22 @@ void RawPointLight::DefaultSetting() {
 	light_.decay = 1;
 }
 
-void RawPointLight::Set(const PointLight& pl) {
+void RawPointLight::Set(const std::string& uuid ,const PointLight& pl) {
+    uuid_ = uuid;
     light_ = pl;
 }
 
+void RawPointLight::Save(std::string _path) {
```

Cppcheck (2.10-2)

[style] The function 'Save' is never used.

(unusedFunction)